src/Symfony/Component/Cache/Tests/DataCollector/CacheDataCollectorTest.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Cache\Tests\Marshaller;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Cache\Adapter\TraceableAdapter;
+use Symfony\Component\Cache\DataCollector\CacheDataCollector;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+class CacheDataCollectorTest extends TestCase
+{
+    private const INSTANCE_NAME = 'test';
+
+    private $traceableAdapter;
+
+    public function testEmptyDataCollector()
+    {
+        $statistics = $this->getCacheDataCollectorStatisticsFromEvents([]);
+
+        $this->assertEquals($statistics[self::INSTANCE_NAME]['calls'], 0, 'calls');
+        $this->assertEquals($statistics[self::INSTANCE_NAME]['reads'], 0, 'reads');
+        $this->assertEquals($statistics[self::INSTANCE_NAME]['hits'], 0, 'hits');
+        $this->assertEquals($statistics[self::INSTANCE_NAME]['misses'], 0, 'misses');
+        $this->assertEquals($statistics[self::INSTANCE_NAME]['writes'], 0, 'writes');
+    }
+
+    public function testOneEventDataCollector()
+    {
+        $traceableAdapterEvent = new \StdClass();
```

Comment:
```suggestion
        $traceableAdapterEvent = new \stdClass();
```

That's how the class is defined in PHP, see https://3v4l.org/4X1As for instance.

Reply:
Good catch! fixed √

Reply:
There are two more occurrences. 😃

Reply:
Haha, tried to push on symfony/symfony instead of my fork 😅  Fixed in the whole file. Thanks